plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -749,6 +750,20 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                 principalPrivileges);
     }
 
+    public static Table makeTable(ConnectorSession session, SchemaTableName schemaTableName, String location, Path targetPath, boolean isExternal)
```

Comment:
I would rename to `buildTable` as make sounds creating a table in metastore or somewhere.